api/database/migrations/2024_04_18_210339_job_alert_default_ignored.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        DB::statement(
+            <<<'SQL'
+            ALTER TABLE users
+            ALTER COLUMN ignored_email_notifications
+            SET DEFAULT '["JOB_ALERT"]'::jsonb
+            SQL, []
+        );
+
+        DB::table('users')->update([
+            'ignored_email_notifications' => ['JOB_ALERT'],
+        ]);
```

Comment:
Given no one should have anything updated from the default as the page for it isn't even merged yet https://github.com/GCTC-NTGC/gc-digital-talent/pull/9971
Can take the simple route?

Reply:
This makes me squirm a bit, since it won't work for any future notification types we want to make default, but yes, this seems safe this time.

Reply:
Yeah, in the future we can't just rewrite the field for everyone 😛